Dockerfile

```diff
@@ -98,6 +98,14 @@ RUN echo 'source("https://bioconductor.org/biocLite.R")' > /opt/packages.r && \
     Rscript /opt/packages.r && \
     mkdir /usr/local/lib/R/site-library
 
+# Install HTSeq
+RUN curl -fsSL https://pypi.python.org/packages/46/f7/6105848893b1d280692eac4f4f3c08ed7f424cec636aeda66b50bbcf217e/HTSeq-0.7.2.tar.gz -o /opt/HTSeq-0.7.2.tar.gz && \
```

Comment:
No, their docs are not very clear. But the download address is PyPI (Python Package Index) and any package there should be installable with `pip`. It _should_ automatically install `numpy` as a dependency, but for whatever reason when I tested it the `htseq` installation instead just complained that `numpy` wasn't installed. The sequential installation seems to do the job though..